packages/fe-fpm-writer/src/common/types.ts

```diff
@@ -32,6 +32,10 @@ export interface CustomElement extends WriterConfig {
      * If set to true then all code snippets are generated in Typescript instead of Javascript.
      */
     typescript?: boolean;
+    /**
+     * If page building block title is defined then it will be used to automatically add a macro page to the generated custom page.
+     */
+    pageBuildingBlockTitle?: string;
```

Comment:
`CustomElement` is used by multiple custom extensions like CustomColumn, CustomSection, CustomAction.
If intention is to add it only for custom page, then please move it to `CustomPage` interface https://github.com/SAP/open-ux-tools/blob/11682ca3f93572d4da5f2572f1ceed469b0fccfb/packages/fe-fpm-writer/src/page/types.ts#L105